tools/docs/build_docs.py

```diff
@@ -49,31 +49,46 @@
 
 flags.DEFINE_string(
     'git_branch',
-    default='master',
+    default=None,
     help='The name of the corresponding branch on github.')
 
+flags.DEFINE_string("output_dir", "/tmp/addons_api",
+                    "Where to output the docs")
+
+CODE_PREFIX_TEMPLATE = "https://github.com/tensorflow/addons/tree/{git_branch}/tensorflow_addons"
 flags.DEFINE_string(
-    'output_dir',
-    default='docs/api_docs/python/',
-    help='Where to write the resulting docs to.')
+    "code_url_prefix", None,
+    "The url prefix for links to the code.")
+
+flags.DEFINE_bool("search_hints", True,
+                  "Include metadata search hints in the generated files")
 
+flags.DEFINE_string("site_path", "addons/api_docs/python",
+                    "Path prefix in the _toc.yaml")
+
+flags.mark_flags_as_mutual_exclusive(['code_url_prefix', 'git_branch'])
 
 def main(argv):
     if argv[1:]:
         raise ValueError('Unrecognized arguments: {}'.format(argv[1:]))
 
-    code_url_prefix = ('https://github.com/tensorflow/addons/tree/'
-                       '{git_branch}/tensorflow_addons'.format(
-                           git_branch=FLAGS.git_branch))
-
+    
+    if FLAGS.git_branch:
```

Comment:
nit: perhaps we could check code_url_prefix first?

Reply:
Yes.